app/src/main/java/org/mozilla/focus/search/MultiselectSearchEngineListPreference.java

```diff
@@ -48,4 +52,36 @@ protected void updateDefaultItem(CompoundButton defaultButton) {
         }
         return engineIdSet;
     }
+
+    // Whenever an engine is checked or unchecked, we notify the menu
+    public void bindEngineCheckboxesToMenu() {
```

Comment:
Maybe rename this to bindOnEngineCheckboxesChanged now that we're not binding to the menu any more, to remove confusion.